packages/stark-core/.eslintrc.json

```diff
@@ -0,0 +1,32 @@
+{
+  "extends": "../../.eslintrc.json",
+  "ignorePatterns": ["!**/*"],
+  "overrides": [
+    {
+      "files": ["*.ts"],
+      "excludedFiles": ["*.spec.ts"],
```

Comment:
Because we override rules for the files : [".ts"] all file [".ts"] are scan. those rule must be override because the prefix for component and directive id defined in this file. to avoid error in test where we create dummy components we should add "excludedFiles": ["*.spec.ts"]